src/modules/integrator-guide/pages/compose-configuration/duplication-detection.adoc

```diff
@@ -5,10 +5,122 @@ include::ROOT:partial$variables.adoc[]
 {PRODUCT_NAME} through the duplicate detection feature helps maintain clean and accurate data by preventing record duplication.
 This is achieved by using rules to apply a criterion that determines how closely a field on a new or edited record matches the same field on an existing record.
 
-The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module thereby preventing duplicates.
+The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module, thereby preventing duplicates.
 
 .Benefits of corteza duplicate detection:
 * Maintain data quality,
 * improves accuracy and efficiency in data,
 * enhances effective description of records,
 * optimizes data storage.
+
+== Validation Rules
```

Comment:
Read through these two bits and decide on one (one is based on what you have, one is based on how I'd (probably) do it) -- both are fine with an obvious bias towards my option :see_no_evil: 

### Express the two types of de-duplication rules more (me version)

Instead of having a `## Validation Rules` section, I'd have two sections (one for each type) -- so `## Soft Duplicate detection` and `## Strict duplicate Detection`.

Each one of these two sections would firstly describe the type and then how to configure it.
A rough sketch:

```adoc
## Soft duplicate detection

* allows duplicates
* warning popup & forum message

### Configuring soft Duplicate Detection

1. go here
2. go there
3. click
4. ...

## Strict duplicate detection

* does not allow duplicates
* error popup & forum message

### Configuring Strict Duplicate detection

1. go here
2. go there
3. click
4. ...

```

I would do this just so that we have two separate configuration sections (in case we'll need to expand them) and give a bit more separation overall.

### Tweak the structure (@Njukierick version)

You outline the two types in an un ordered list which, when rendered, looks like this:

![image](https://user-images.githubusercontent.com/12545711/196206451-56ee7ef0-f5a3-4619-95e5-1eb5bf85cbca.png)

That is because of the empty lines you have in there.

What we've put in place, when describing options with longer descriptions (such as a workflow step type, a de-duplicate detection rule type, ...), is we present it either in sub-sections or in a table which looks like this ([src](src/modules/integrator-guide/pages/dal-connections/index.adoc)):

```adoc
[cols="1s,5a"]
|===
| [#anchor-id]#<<anchor-id,Label>>#
|
Text here
|===
```

What this table does is it removes the need for sub-sections (if we're talking about something smaller) while preserving structure and providing anchor links to send users directly to the paragraph

![image](https://user-images.githubusercontent.com/12545711/196207555-b83e838f-a842-49e1-9849-e4957738c392.png)